README.md

```diff
@@ -7,12 +7,6 @@
 Puppet Lint tests Puppet code against the recommended [Puppet language style
 guide](http://puppet.com/docs/puppet/latest/style_guide.html). Puppet Lint validates only code style; it does not validate syntax. To test syntax, use Puppet's `puppet parser validate` command.
 
-## Compatibility warning
```

Comment:
This is useful information. Please keep it in. Perhaps update it to make it clear it is current.